app.js

```diff
@@ -1,7 +1,15 @@
 const express = require('express');
-const { getTopics } = require('./controllers/news.controllers');
+const { getTopics, getArticles, getArticle } = require('./controllers/news.controllers');
 const app = express();
 
 app.get('/api/topics', getTopics)
+app.get('/api/articles', getArticles)
+app.get('/api/articles/:article_id', getArticle)
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) res.status(err.status).send({msg: err.msg})
+	else if (err = '22P02') res.status(400).send({ msg: "Doesn't exist"})
```

Comment:
it looks like you're only using a single `=` rather than `===` - I am surprised this didn't give you an error ?